src/test/java/test_cases/LoginTests.java

```diff
@@ -0,0 +1,63 @@
+package test_cases;
+
+import common.helpers.BrowserHelper;
+import common.helpers.Constant;
+import common.helpers.DataHelper;
+import model.User;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.HomePage;
+import page_objects.LoginPage;
+
+public class LoginTests extends BaseTest {
+  HomePage homePage = new HomePage();
+  LoginPage loginPage = new LoginPage();
+
+  @Test(description = "User can log into Railway with valid username and password")
+  public void TC01() {
+    User user = new User(Constant.USERNAME, Constant.PASSWORD);
+
+    homePage.clickLoginTab();
+    loginPage.login(user);
+
+    String actualResult = homePage.getWelcomeMessage();
+    String expectedResult = "Welcome " + user.getUsername();
+
+    Assert.assertEquals(actualResult, expectedResult);
```

Comment:
there should be failed message for all check point to describe what we're verifying

Reply:
Thanks for letting me know,  I added message for all check point